lib/components/table.vue

```diff
@@ -524,14 +533,25 @@
 
                 if (data.then && typeof data.then === 'function') {
                     // Provider returned Promise
-                    data.then((items) => {
+                    data.then(items => {
                         this._providerSetLocal(items);
                     });
                 } else {
                     // Provider returned Array data
                     this._providerSetLocal(data);
                 }
+            },
+            hasFormatter(field) {
+                return field.formatter && ((typeof (field.formatter) === 'function') || (typeof (field.formatter) === 'string'));
+            },
+            callFormatter(item, key, field) {
+                if (field.formatter && (typeof (field.formatter) === 'function'))
+                    return field.formatter(item[key]);
+
+                if (field.formatter && (typeof (this.$parent[field.formatter]) === 'function'))
+                    return this.$parent[field.formatter](item[key]);
```

Comment:
then we must pass these vars from child component. 
and this declaration won't work, as parent functions aren't in children's scope. in case we want to use parent's methods, we must supply it as String, Function declaration only for global scoped which will be available to child
```
data: {
    fields: {
      foo: { sortble: true, formatter this.fooFormatter },
      bar: { sortble: true, formatter this.barFormatter }
    }
  },
```